fastlane/Pluginfile

```diff
@@ -0,0 +1,5 @@
+# Autogenerated by fastlane
+#
+# Ensure this file is checked in to source control!
+
+gem 'fastlane-plugin-release_actions', git: 'https://github.com/aws-amplify/amplify-ci-support', branch: 'master', glob: 'src/fastlane/release_actions/*.gemspec'
```

Comment:
Hmm, should we lock to a SHA?

Reply:
It is in the Gemfile.lock: https://github.com/aws-amplify/amplify-ios/blob/2edfd7bcbe44a68bf5b5f1d94415bb947131fe53/Gemfile.lock#L3

I didn't want to set a `ref` in the Pluginfile in order to allow updating easily with `bundle update` as we release new revisions of the plugin

Reply:
My concern is if we update master with a breaking change, that would break the pipelines. Since it's a manual action to bundle, and this isn't a ruby project, it should be fine.

Reply:
Ignore, discussed offline, this only happens on `bundle update`.